docs/sources/alerting/alerting-rules/alerting-migration/_index.md

```diff
@@ -66,9 +66,13 @@ To convert data source-managed alert rules to Grafana managed alerts:
 
 1. Navigate to the Data source-managed alert rules section and click **Import to Grafana-managed rules**.
 
-   The import alert rules page opens.
+   Alternately, you can click **More > Import alert rules** to open the import rules page.
```

Comment:
This is only for instance with the `alertingListViewV2` enabled. Not sure how to address this difference here in docs? @konrad147 wdyt?

Reply:
I think we should remove it for now. We'll update the docs when we switch to the new list for most customers